code/game/jobs/job/outsider/representative.dm

```diff
@@ -85,8 +76,34 @@
 
 	return rep_objectives
 
+/datum/job/consular
```

Comment:
This is missing the species restrictions.

Reply:
how does this behave with contractors, btw? i assume as expected

Reply:
yeah, contractors cannot choose this option. you have to set your faction to unaffiliated to have this job as an option